### DISCUSSION TITLE
"Exclude instead" functionality

### DISCUSSION THREAD

**Discussion by andriAntt** [algolia/instantsearch#5354] [Q&A] [answered]
I want to create select component with React-Instantsearch-hooks which would work similarly to Searchable RefinementList component, but without checkboxes. But I also need "Exclude instead" button.  It should hide all hits with value that was clicked with "Exclude instead" button. Something like "not" toggle. I've searched docs but haven't found any solutions. I think I can refine all values except excluded one, but it is not the best solution, because I also have CurrentRefinements component that would be overfilled in such a case (more than 50 available values in one filter).

Could you advise any solution?

**Comment by Haroenv** [comment, marked as answer]
The UI for exclusion lists is hard to get right, and when we researched this a while ago, couldn't find an implementation that did it right. Therefore this isn't implemented in the library. However, there's an example here on how to make negative refinement list here: https://github.com/Haroenv/instantsearch-negative-refinement-list
